enhancements/console/quict-starts.md

```diff
@@ -0,0 +1,143 @@
+
+## Motivation
+
+Help users with their understanding the principles of their workflows by
+guiding them though the necessary steps.
+
+### Goals
+
+1. Provide a mechanism to display user guides for various workflows.
+2. Make users understand what are the steps needed to achieve their goals.
+3. Provide new CRD format for writing the Quick Starts.
+4. Have a repository for the out-of-the-box Quick Starts, those that describe how to install an operator or go though standard workflow.
+
+## Proposal
+
+* Introduce CRD format that will be used for writing Quick Starts.
+* Introduce default repository for Quick Starts.
+  
+### User Stories
+
+#### Story 1
+
+As an administrator of an OpenShift cluster, I need a guide to walk me through how to install OpenShift Serverless modules (Serving and Eventing) in a cluster.
+
+#### Story 2
+
+As an administrator of an OpenShift cluster, I need a guide to walk me through how to update an OpenShift cluster.
+
+#### Story 3
+
+As a developer, I need a guide to walk me through how to deploy an existing application as a serverless workload.
+
+#### Story 4
+
+As a operator creator I want to provide operator consumers with a guide on how to install and user the my operator.
+
+### Implementation Details
+
+1. In order to provide mechanism to discribe a Quick Start, new CRD named `QuickStarts` will be created.
+2. A new `openshift/quick-tours` repository will be created which will contain all supported Quick Starts. The `openshift/quick-tours` repository will have `release-*` branches.
+3. `console-operator` will import all the existing Quick Starts CRs from appropriate branch of the `openshift/quick-starts` repository into the `/manifest` directory, so that the CVO can:
+   * create them if the CR doesn't exists.
+   * update the CRs upon the cluster update (since CVO is doing `apply`).
+4. `console-operator` will be resposible for removing older `QuickStarts` versions, based on their CRs `spec.version` field that will be in semver format. If the semver version range won't match the cluster version the `console-operator` will remove the CR.
+5. Steps in the Quick Starts will support basic markdown thats already in use in the OpenShift's Console. 
+6. All Quick Starts will be listed available in a separate page that will be accessible from Help Menu.
+   ![help-menu](https://raw.githubusercontent.com/jhadvig/images/master/help-menu.png)
+7. For different areas of interest, the links for Quick Start will need to be handles separately:
+   * Operators - CSV of the operator that has a Quick Starts published will contain `console.openshift.io/quick-start: <quick-start-name>` annotation. Upon the operator installation an "Quick Start" link will appear next to the `Install` button.
+   ![operators](https://raw.githubusercontent.com/jhadvig/images/master/operators.png)
+   * ...TBD
+
+
+QuickStarts CR for [Explore Serverless](https://marvelapp.com/236ge4ig/screen/69908905):
```

Comment:
What would the full version of this look like? The tour content is truncated.

Reply:
Here is full version https://github.com/openshift/console/pull/5983/files

Reply:
@jhadvig Thanks! Has thought been given to externalizing the tour content specifically, so that it might be managed kind of like CCS' [modularized documentation](https://github.com/openshift/openshift-docs/tree/master/modules)?